examples/simple_zonal_with_hub/hub.tf

```diff
@@ -17,6 +17,7 @@
 module "hub" {
   source           = "../../modules/hub"
   project_id       = var.project_id
+  hub_project_id   = var.hub_project_id
```

Comment:
nit: since we are not using another hub project in the examples, we can drop this here and throughout.

Reply:
Thanks, I'll remove the `hub_project_id` reference from each of the examples `hub.tf` and `variables.tf`  files.